src/cipher/address_test.go

```diff
@@ -1,24 +1,77 @@
 package cipher
 
 import (
+	"math/rand"
 	"testing"
-
-	"github.com/stretchr/testify/require"
+	"time"
 
 	"github.com/skycoin/skycoin/src/cipher/base58"
+	"github.com/stretchr/testify/require"
 )
 
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+// Generate random string of length n
+func RandStringBytes(n int) []byte {
+	b := make([]byte, n)
+	for i := range b {
+		b[i] = base58.BTCALPAHBET[rand.Intn(len(base58.BTCALPAHBET))]
+	}
+
+	return b
+}
+
```

Comment:
We should pull all addresses from blockchain and save them to a file, and use that as test data for a test of the skycoin address parser